util/strings.go

```diff
@@ -94,3 +94,18 @@ func CoalesceString(in ...string) string {
 func CoalesceStrings(inArray []string, inStrs ...string) string {
 	return CoalesceString(CoalesceString(inArray...), CoalesceString(inStrs...))
 }
+
+// StringContainsSliceRegex determines if a string matches a regex in a slice
+func StringContainsSliceRegex(slice []string, item string) bool {
+	if len(slice) == 0 {
```

Comment:
we don't really need to check this; it's implied by 110 since we just skip the for loop